content/admin/manage_orgs/Licensing.md

```diff
@@ -4,51 +4,53 @@ description: "Overview of Licensing visibility"
 tags: "Admin manage SuperAdmin EOP SaaS RASP IAST Licensing"
 -->
 
-## Granting Licenses
-A SuperAdmin may grant separate Assessment (IAST) and Protection (RASP) licenses to an organization via the **Organizations** page. Assessment licenses apply to applications while Protection licenses apply to servers. The **Licenses** column will list the number of used and unused licenses that have been granted for both types and a red warning icon will be displayed if licenses are nearing expiration.  
+Allocate, revoke and manage Assess and Protect licenses for individual organizations as a SuperAdmin. An Organization Admin may then apply and remove Protect licenses for servers in their organization. 
 
-<a href="assets/images/Licensing_Grid.png" rel="lightbox" title="SuperAdmin Organizations Page"><img class="thumbnail" src="assets/images/Licensing_Grid.png"/></a>
+## Supervise Licenses
 
-Select **Grant Licenses** from the organization's row menu dropdown.  The dialog that follows will allow you to choose the number of licenses as well as the desired expiration date if you are using Contrast's SaaS product.
+A SuperAdmin may grant separate Assess and Protect licenses to an organization in the **Organizations** page. (Assess licenses apply to applications and Protect licenses apply to servers.) The **Licenses** column in the organizations grid lists the number of used and unused licenses that have been granted. If licenses is nearing expiration, a red warning icon appears in the grid row. 
 
-In the case of EOP, expiration dates will already be set, as will the total number of licenses available to grant.  These counts will show in the dialog.
+<a href="assets/images/Org-grid.png" rel="lightbox" title="Organizations page"><img class="thumbnail" src="assets/images/Org-grid.png"/></a>
 
-<a href="assets/images/Licensing_Grant.png" rel="lightbox" title="Granting A License (SaaS)"><img class="thumbnail" src="assets/images/Licensing_Grant.png"/></a>
+### Allocate Licenses
 
+To grant licenses to an organization, click the arrow in the grid row and select **Allocate Licenses** from the dropdown menu. In the dialog that appears, SaaS users can enter the number of Assess and/or Protect licenses to allocate to that organization in the given fields. Click in the corresponding **Expiration Date** fields to make your selections on the calendars. EOP users can choose to allocate licenses from the total number available, which is shown in dialog along with the predetermined expiration dates. 
```

Comment:
I would revise to:  
"To grant licenses to an organization, select **Allocate Licenses** from the row action dropdown menu. In the dialog that appears, SaaS users can enter the number of Assess and/or Protect licenses they wish to allocate to that organization, as well as define their respective expiration dates. EOP users can choose to allocate licenses from the total number available, which is displayed in the dialog along with predetermined expiration dates."